packages/test-runner-visual-regression/src/visualDiffCommand.ts

```diff
@@ -100,7 +100,9 @@ export async function visualDiffCommand(
 
   return {
     errorMessage: !passed
-      ? `Visual diff failed. New screenshot is ${diffPercentage.toFixed(2)} % different.`
+      ? `Visual diff failed. New screenshot is ${diffPercentage.toFixed(
+          2,
+        )}% different.\nSee diff for details: ${resolveImagePath(baseDir, diffName)}`
```

Comment:
This looks like doing the same work twice. How about changing the method above?

```js
const saveDiff = async () => {
  const diffPath = resolveImagePath(baseDir, diffName);
  await options.saveDiff({
    filePath: diffPath,
    baseDir,
    name: diffName,
    content: diffImage,
  });
  return diffPath;
};
```

Then we can declare the variable above:

```js
let diffPath
```

And update both calls accordingly:

```js
diffPath = await saveDiff();
```

So that we can later include this into the error message.

Reply:
Sounds good to me. Will update tomorrow

Reply:
Thanks! And then please aldo add xhangeset using `npx changeset` as described in the above comment.

Reply:
Done! Not exactly as you described, but it has same result (I calculate it outside of `saveDiff` to avoid changing function signatures). I also added a changeset and rebased/squashed

Reply:
Thanks! Your approach does make sense and looks better than what I suggested 👍